python/ccxt/bitz.py

```diff
@@ -281,21 +280,26 @@ def fetch_trades(self, symbol, since=None, limit=None, params={}):
         trades = response['data']['d']
         return self.parse_trades(trades, market, since, limit)
 
-    def parse_ohlcv(self, ohlcv, market=None, timeframe='1m', since=None, limit=None):
-        volume = ohlcv[5] if self.options['fetchOHLCVVolume'] else None
-        return [
-            ohlcv[0],
-            ohlcv[1],
-            ohlcv[2],
-            ohlcv[3],
-            ohlcv[4],
-            volume,
-        ]
+    def parse_ohlcvs(self, ohlcvs, market=None, timeframe='1m', since=None, limit=None):
```

Comment:
I don't really get this edit:

- replaced parse_ohlcv method with parse_ohlcvs method for some reason, effectively deleting the warning as well
- removed the parse_ohlcv definition, though it was fine
- now calling the base parse_ohlcv, which does not cover the volume which was the subject of the warning
- deleted the warning as well for some reason
- copied base parse_ohlcvs method, no need to reinvent the wheel in this particular place

Sorry, but I can't really merge it, because I see no point in these edits the way they are. We should have at least saved the existing functionality and the warning. Just because you didn't get it doesn't mean you have to edit it right away. Moreover, all of the above edits are done without taking the CONTRIBUING doc into account... Hope this does not stop you from committing in the future. Sorry once again, and thx for the effort anyways.

Reply:
Ignore the above comment, plz.